apps/veil/src/pages/portfolio/ui/asset-row.tsx

```diff
@@ -0,0 +1,290 @@
+                  />
+                ) : (
+                  <Text variant={'smallTechnical'} color='text.secondary'>
+                    -
+                  </Text>
+                )}
+              </div>
+            ) : (
+              <Text variant={'smallTechnical'} color='text.secondary'>
+                Cosmos wallet not connected
+              </Text>
+            )}
+          </TableCell>
+          <TableCell variant={variant}>
+            {price ? (
+              <div className='flex flex-col'>
+                <Text variant={'smallTechnical'} color='text.secondary'>
+                  {price.price.toFixed(4)} {price.quoteSymbol}
+                </Text>
+              </div>
+            ) : (
+              <Text variant={'smallTechnical'} color='text.secondary'>
+                -
+              </Text>
+            )}
+          </TableCell>
+          <TableCell variant={variant}>
+            {shieldedValue > 0 && price ? (
+              <Text variant={'smallTechnical'} color='text.secondary'>
+                {shieldedValue.toFixed(2)} {price.quoteSymbol}
+              </Text>
+            ) : (
+              <Text variant={'smallTechnical'} color='text.secondary'>
+                -
+              </Text>
+            )}
+          </TableCell>
+          <TableCell variant={variant}>
+            {publicValue > 0 && price ? (
+              <Text variant={'smallTechnical'} color='text.secondary'>
+                {publicValue.toFixed(2)} {price.quoteSymbol}
+              </Text>
+            ) : (
+              <Text variant={'smallTechnical'} color='text.secondary'>
+                -
+              </Text>
+            )}
+          </TableCell>
+          <TableCell variant={variant}>
+            {totalValue > 0 && price ? (
+              <Text variant={'smallTechnical'} color='text.secondary'>
+                {totalValue.toFixed(2)} {price.quoteSymbol}
+              </Text>
+            ) : (
+              <Text variant={'smallTechnical'} color='text.secondary'>
+                -
+              </Text>
+            )}
+          </TableCell>
+        </div>
+        {isExpanded &&
+          asset.publicBalances.map(bal => (
+            <div key={bal.denom} className={`col-span-6 grid grid-cols-subgrid`}>
+              <TableCell variant={'lastCell'}>
+                <div className='flex items-center'>
+                  <Text variant={'smallTechnical'} color='text.secondary'></Text>
+                </div>
+              </TableCell>
+              <TableCell variant={'lastCell'}>
+                <div className='flex items-center gap-3 justify-between w-full'>
+                  <div className={'py-3'}>
+                    <ValueViewComponent
+                      valueView={bal.valueView}
+                      trailingZeros={false}
+                      priority={'tertiary'}
+                      density={'slim'}
+                      context={'table'}
```

Comment:
question/nitpick: i wonder how did you start using this style of attribute syntax? I mean, didn't see you using it before, but here's clearly many attributes are written like this. Not all though, so it's a mix

Reply:
I write some of my code with AI, so maybe that's why. Is there a preferred style you would prefer I use?

Reply:
the preferred style is any style that's consistent throughout the codebase. I'd say curly braces are only needed when you want to pass a variable in them